openedx/features/qverse_features/registration/forms.py

```diff
@@ -22,3 +26,57 @@ class Meta(object):
                 'field_type': 'select'
             }
         }
+        error_messages = {
+            'registration_number': {
+                'required': 'Please enter your Registration Number.',
+            },
+            'first_name': {
+                'required': 'Please enter your First Name.',
+            },
+            'surname': {
+                'required': 'Please enter your Surname.',
+            },
+            'current_level': {
+                'required': 'Please select a valid Current Level.'
+            },
+            'department': {
+                'required': 'Please select a valid Department'
+            },
+            'programme': {
+                'required': 'Please select the valid Programme that you have been enrolled in.'
+            },
+        }
+        help_texts = {
+            'registration_number': 'Enter your registration number provided from your school.',
```

Comment:
```suggestion
            'registration_number': 'Enter your registration number provided by your school.',
```
Just a suggestion. Please ignore if doesn't seem valid.

Reply:
![image](https://user-images.githubusercontent.com/42166091/79560861-cfb4d680-80c1-11ea-95df-4e496f6e2741.png)
This is the actual suggestion that I gave. If github isn't rendering it correctly.